File: JavaScript/leetcode75/src/test/Problem07_ProductOfArrayExceptSelf.test.js
const productExceptSelf = require('../main/Problem07_ProductOfArrayExceptSelf');

test('should product Except Self', () => {
    const nums = [1,2,3,4];
    const expected = [24,12,8,6];
    expect(productExceptSelf(nums)).toStrictEqual(expected);
});

File: Java/leetcode75/src/test/java/org/phoenix/leetcode/challenges/Problem07_ProductOfArrayExceptSelfTest.java
package org.phoenix.leetcode.challenges;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertArrayEquals;

class Problem07_ProductOfArrayExceptSelfTest {

    private final Problem07_ProductOfArrayExceptSelf test = new Problem07_ProductOfArrayExceptSelf();

    @Test
    void productExceptSelf() {
        int[] nums = new int[]{1,2,3,4};
        int[] expected = new int[]{24,12,8,6};
        assertArrayEquals(expected, test.productExceptSelf(nums));
    }
}